pom.xml

```diff
@@ -130,9 +131,10 @@
         <com.google.apis.storage.version>v1-rev20230301-2.0.0</com.google.apis.storage.version>
         <jdk.strong.encapsulation.argLine><!-- empty placeholder --></jdk.strong.encapsulation.argLine>
         <jdk.security.manager.allow.argLine><!-- empty placeholder --></jdk.security.manager.allow.argLine>
-        <repoOrgId>maven.org</repoOrgId>
-        <repoOrgName>Maven Central Repository</repoOrgName>
-        <repoOrgUrl>https://repo1.maven.org/maven2/</repoOrgUrl>
+        <repoOrgId>apache.snapshots</repoOrgId>
+        <repoOrgName>Apache Snapshot Repository</repoOrgName>
+        <repoOrgUrl>https://repository.apache.org/snapshots</repoOrgUrl>
```

Comment:
why are we changing this?

Reply:
I don't know why we are changing this but it's similar to what we have in 25.0.0-confluent. https://github.com/confluentinc/druid/blob/adf75bc46226cbf99be27c4a14dd65d6db6f7dff/pom.xml#L127
@m-ghazanfar, do you know?

Reply:
Reverted this change